book/src/keymap.md

```diff
@@ -320,6 +320,10 @@ Mappings in the style of [vim-unimpaired](https://github.com/tpope/vim-unimpaire
 | `]t`     | Go to previous test (**TS**)                 | `goto_prev_test`      |
 | `]p`     | Go to next paragraph                         | `goto_next_paragraph` |
 | `[p`     | Go to previous paragraph                     | `goto_prev_paragraph` |
+| `]g`     | Go to next change                            | `goto_next_change`    |
+| `[g`     | Go to previous change                        | `goto_prev_change`    |
+| `]G`     | Go to first change                           | `goto_first_change`   |
+| `[G`     | Go to last change                            | `goto_last_change`    |
```

Comment:
I have some reservations about the need for capital versions of textobjects (like `]D`/`[D`): you could emulate their behavior with `gg]<the textobject>`/`ge[<the textobject>` or create macros for them (also see https://github.com/helix-editor/helix/issues/4561). What do you think @archseer?

Reply:
I tried to just match the existing bindings for diagnostics here to ensure the bindings were consistent. I was not aware of previous discussion in that regard. I can see that a seperate command is not really necessary here although one nice advantage of habing a seperate command for this is s that `]G` does not jump if the file is unchanged (same as with `]D` then there are no diagnostics) which can be useful. Just let me know what I should do here

Reply:
That sounds reasonable, I will leave them in for consistency for now and we can remove them again in the future 👍